controllers/configurationpolicy_controller.go

```diff
@@ -766,17 +909,37 @@ func (r *ConfigurationPolicyReconciler) handleObjectTemplates(plc *policyv1.Conf
 	}
 
 	parentStatusUpdateNeeded := false
+	var tmplResolver *templates.TemplateResolver
+	var resolveOptions *templates.ResolveOptions
+
+	relatedObjects := []policyv1.RelatedObject{}
 
 	if !disableTemplates {
-		updateNeeded, tmplErr := r.handleTemplatization(plc, usingWatch)
-		if tmplErr != nil {
-			return tmplErr
+		var err error
+
+		tmplResolver, resolveOptions, err = r.getTemplateResolver(plc)
+		if err != nil {
+			return err
 		}
 
-		parentStatusUpdateNeeded = updateNeeded
+		if plc.Spec.ObjectTemplatesRaw != "" {
+			err := r.resolveObjectTemplatesRaw(plc, tmplResolver, resolveOptions)
+			if err != nil {
+				return err
+			}
+
+			// Templates are already handled so disable any further processing.
+			disableTemplates = true
```

Comment:
Nice--I was wondering about the use of the namespace template variable with `object-templates-raw`, but I think this makes sense to just shut it off after processing it.